docm_filter.pl

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use feature qw(say);
+
+die("wrong number of arguments") unless scalar(@ARGV) == 2;
+my ($docm_out_vcf, $outdir) = @ARGV;
+
+open(my $docm_vcf_fh, $docm_out_vcf)
+    or die("couldn't open $docm_out_vcf to read");
+open(my $docm_filter_fh, ">", "$outdir/docm_filter_out.vcf")
+    or die("couldn't open docm_filter_out.vcf for write");
+
+while (<$docm_vcf_fh>) {
+    chomp;
+    if (/^#/) {
+        say $docm_filter_fh $_;
+    }
+    else {
+        my @columns = split /\t/, $_;
+        my @tumor_info = split /:/, $columns[10];
```

Comment:
At least so far it is consistent for GATK Haplotypecaller 3.6, 3.7 command test if "-I normal.bam -I tumor.bam" (normal in front of tumor) is used in command line. GATK_haplotype_caller.cwl assures normal first tumor second.